### PR TITLE
UI bug fixes 20 05 2022

### PR DESCRIPTION
**Pull request by AshleyRedmanJisc**
Fixes to 3 open issues (see issue list to the right).

### CODE REVIEW

File: ui/src/pages/publications/[id]/flag/[flagId]/index.tsx [hunk 1]
```diff
@@ -89,6 +89,8 @@ const FlagThread: Next.NextPage<Props> = (props): JSX.Element => {
         }
     );
 
+    console.log(data);
```
Comment (tomgiddingsjisc) [inline ui/src/pages/publications/[id]/flag/[flagId]/index.tsx]: This can probably be removed?

**Review by tomgiddingsjisc — COMMENTED** [review]
Just the one stray console log otherwise looks great!

File: ui/src/pages/publications/[id]/flag/[flagId]/index.tsx (hunk 1 above)
Comment (AshleyRedmanJisc) [reply ui/src/pages/publications/[id]/flag/[flagId]/index.tsx]: Removed in the last commit

**Review by tomgiddingsjisc — APPROVED** [review]
LGTM!

**Pull request merged**